tests/parser.rs

```diff
@@ -0,0 +1,7 @@
+use nds::parser::NDSParser;
+use std::convert::TryFrom;
+
+#[test]
+fn test_parsing() {
+    assert!(NDSParser::try_from("test_nds_files/mario_kart_ds.nds").is_ok());
```

Comment:
Ideally these tests should be on files that can be uploaded to github without copyright issues, although I know that may not be super easy to find. Not a blocker if the test doesn't work since other files are missing too.

Reply:
Alright, I'll try to find some `.nds` files for us. :) But let me fix some stuff first please.

Reply:
Do you have any tipps for searching some `.nds` rom files without any issues for us? Do I just have to look after the license?

Reply:
I don't know where would be best, but I'm sure there are some homebrew games that are free to use. I think the small and big files I was using before were also generic, but unsure. One other option is to sort of bootstrap the tests by using the library to build an nds file, but that runs into potential issues where if it's wrong now it'll not be factored into the test because we'd be testing against it assuming it's right.